proposals/N3GOV-43.json

```diff
@@ -0,0 +1,42 @@
+{
+  "content": {
+    "kind": "treasury-management",
+    "calls": [
+      {
+        "chainId": 1,
+        "from": "0x336252602b3a8a0be336ed942228305173e8082b",
+        "to": "0x5CB5Ba62DC8ced477f20d4692986f1Cd42174761",
+        "method": "release(address)",
+        "parameters": [
+          "0xa117000000f279D81A1D3cc75430fAA017FA5A2e"
+        ]
+      },
+      {
+        "chainId": 1,
+        "from": "0x336252602b3a8a0be336ed942228305173e8082b",
+        "to": "0xa117000000f279D81A1D3cc75430fAA017FA5A2e",
+        "method": "transfer(address,uint256)",
+        "parameters": [
+          "0x7fFf737ae9FdDD6E28Fe8B8F0F4dE69Bb8e7a885",
+          "20000000000000000000000"
+        ]
+      },
+      {
+        "chainId": 1,
+        "from": "0x336252602b3a8a0be336ed942228305173e8082b",
+        "to": "0x333A4823466879eeF910A04D473505da62142069",
+        "method": "transfer(address,uint256)",
+        "parameters": [
+          "0x7fFf737ae9FdDD6E28Fe8B8F0F4dE69Bb8e7a885",
+          "69000000000000000000"
+        ]
+      }
+    ]
+  },
+  "id": 1337,
+  "spec": 2,
+  "votes": [
+    null
+  ],
```

Comment:
@gcattinp I think there is a bug with the code generated by the [Proposal Generator](https://gov.nation3.org/proposals/create), so try removing the `votes` property:

```suggestion
```